### PR TITLE
Add optional timestamp argument to easyrtc.string()

### PR DESCRIPTION
**Pull request by renzenicolai**

### CODE REVIEW

File: esp32/modules/easyrtc.py [hunk 1]
```diff
@@ -7,8 +7,11 @@
 import machine, time
 
 # Functions
-def string(date=False, time=True):
-    [year, month, mday, wday, hour, min, sec, usec] = machine.RTC().datetime()
+def string(print_date=False, print_time=True, timestamp = -1):
+    if timestamp<0:
+        [year, month, mday, wday, hour, minute, sec, usec] = machine.RTC().datetime()
+    else:
+        [year, month, mday, hour, minute, second, wday, yday] = time.localtime(timestamp)
```
Comment (raboof) [inline esp32/modules/easyrtc.py]: shouldn't `second` be `sec` here (or vice-versa)?

pretty confusing that these 2 assignments assign to different variables, might be clearer to rename `usec` to `unused_usec` and `yday` to `unused_yday` to make explicit that those aren't used?

File: esp32/modules/easyrtc.py [hunk 2]
```diff
@@ -7,8 +7,11 @@
 import machine, time
 
 # Functions
-def string(date=False, time=True):
-    [year, month, mday, wday, hour, min, sec, usec] = machine.RTC().datetime()
+def string(print_date=False, print_time=True, timestamp = -1):
```
Comment (raboof) [inline esp32/modules/easyrtc.py:10]: nice naming improvement, though not strictly speaking backwards compatible. can we be sure no-one was using the old names?

**Review by raboof — CHANGES_REQUESTED** [review]

File: esp32/modules/easyrtc.py (hunk 2 above)
Comment (renzenicolai) [reply esp32/modules/easyrtc.py:10]: Yes, we can be sure.

File: esp32/modules/easyrtc.py (hunk 1 above)
Comment (renzenicolai) [reply esp32/modules/easyrtc.py]: they are completely different functions with a completely different return value...
I agree that sec should be second. Will change that

**Pull request merged**